src/content/en/updates/2016/10/avoid-not-secure-warn.md

```diff
@@ -0,0 +1,64 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: "Chrome will soon mark non-secure pages containing password and credit card input fields as Not Secure in the URL bar."
+
+{# wf_updated_on: 2016-10-24 #}
+{# wf_published_on: 2016-10-24 #}
+{# wf_featured_image: /web/updates/images/generic/images/warning.png #}
+{# wf_featured_snippet: Chrome will soon mark non-secure pages containing password and credit card input fields as Not Secure in the URL bar. This document is intended to aid Web Developers in updating their sites to avoid this warning. #}
+
+# Avoiding the Not Secure Warning in Chrome {: .page-title }
+
+As [announced in September](https://security.googleblog.com/2016/09/moving-towards-more-secure-web.html),
+Chrome will soon mark non-secure pages containing **password** and **credit
+card** input fields as **Not Secure** in the URL bar.
+
+This document is intended to aid Web Developers in updating their sites to avoid
+this warning.
+
+## Enable warnings
+
+Warnings will be enabled by default for everyone in Chrome 56, slated for
+release in January 2017.
+
+To test the upcoming user experience before then, install the latest [Google
+Chrome Canary](https://www.google.com/chrome/browser/canary.html) build. At the
+time of this writing, Chrome Canary is version 56. Version 56.0.2895.0 is the
+first version to include detection of sensitive fields. You can exercise the
+browser’s warning behavior on [this page](http://http-password.badssl.com/).
+
+When the Not Secure state is shown, the DevTools console shows the message `In
+Chrome M56 (Jan 2017), this page will be marked as “not secure” in the URL bar.
+For more information, see [https://goo.gl/zmWq3m](https://goo.gl/zmWq3m).`
```

Comment:
At present, this URL is literally the URL in the product; since we're matching the error message, I'm not sure that changing the link to something else makes sense.